app/views/services/show.html.erb

```diff
@@ -24,12 +24,18 @@
           <p class="card-text"><%= @service.price %>€</p>
 
           <%= link_to services_path do%>
-          <p class="btn btn-danger">Back to Services</p>
+          <p class="btn btn-primary">Back to Services</p>
           <% end %>
 
           <!-- no link for booking new yet so i left a second index path -->
           <%= link_to new_service_booking_path(@service) do%>
-          <p class="btn btn-primary">Book Service</p>
+          <p class="btn btn-success">Book Service</p>
+          <% end %>
+
+          <% if @service.user.id == @user.id %>
+            <%= link_to service_path(@service.id), method: :delete do%>
```

Comment:
can you add a confirmation :
method: :delete,
        data: { confirm: "Are you sure?" }

Reply:
I literally was going to look up how to add a confirm pop-up when I got in today! 

I'l do it now and re-push.

Reply:
done! See next commit.